package.json

```diff
@@ -46,7 +46,7 @@
     "@types/boom": "^7.2.1",
     "@types/console-log-level": "^1.4.0",
     "@types/express": "^4.17.0",
-    "@types/hapi": "^18.0.2",
```

Comment:
Given that these are all dev dependencies, I think `chore` is the right prefix here 👍

Reply:
Gotcha